shared/src/main/java/com/sap/psr/vulas/shared/enums/Scope.java

```diff
@@ -3,21 +3,42 @@
 import java.util.HashSet;
 import java.util.Set;
 
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+
+
+
 /**
  * Use in com.sap.psr.vulas.backend.model.Dependency and com.sap.psr.vulas.shared.json.model.Dependency.
  */
 public enum Scope {
-	COMPILE, PROVIDED, RUNTIME, TEST, SYSTEM, IMPORT;
+	COMPILE, PROVIDED, RUNTIME, TEST, SYSTEM, IMPORT, CUSTOM;
+
+	private static final Log log = LogFactory.getLog(Scope.class);
 	
 	public static final Set<Scope> fromStringArray(String[] _values) {
 		final Set<Scope> scopes = new HashSet<Scope>();
 		if(_values!=null) {
 			for(String v: _values) {
 				try {
```

Comment:
Cant we replace the entire try-catch clause simply by calling `scopes.add(Scope.fromString(v.toUpperCase())`?